tools/asynccmd/asynccmd.go

```diff
@@ -0,0 +1,176 @@
+package asynccmd
+
+import (
+	"os"
+	"os/exec"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/bitrise-io/go-utils/log"
+)
+
+// Status ...
+type Status struct {
+	Code int
+	Err  error
+}
+
+// Cmd ...
+type Cmd struct {
+	Name string
```

Comment:
Why are these fields exported? Is it because the name conflict with the `New` method's parameters? If that is the reason, just use different var names.